### ISSUE TITLE
VR 2FA

### ISSUE THREAD

**Issue by obccgn** [nemiah/phpFinTS#434]
Hello,
is it possible to somehow circumvent the 2FA obligation of Volksbank? It would not be possible to process automated requests (cronjobs) if 2FA is activated, correct? Or is it possible to keep it active generally for security purposes on bank's side but to skip within the PHP script?
Thanks

**Comment by nemiah** [comment]
I don't think it is possible to skip 2FA. What you can do, is remote-control the app via adb :grin:
